### DISCUSSION TITLE
Neper and Abaqus joint problem

### DISCUSSION THREAD

**Discussion by ccis88** [neperfepx/neper#711] [Q&A] [answered]
neper -T -n 1000 -dim 2 -domain "circle(0.018):translate(-0.009,-0.009)"  -morpho voronoi
neper -M n1000-id1.tess -elttype tri -format inp
   The above is my instruction in neper, after the inp file is imported into abaqus, the center of the circle has some deviations from the origin, I don't understand it, thanks for the answer!
![e718f9ced569ee2530c54dbef593a30](https://github.com/neperfepx/neper/assets/122538253/345dab3f-15af-4173-bce3-18951c7ad95d)
The ring in the figure is a ring created in abaqus with the origin of the circle.

**Comment by rquey** [comment, marked as answer]
Thanks for the bug report.  Fixed in version 4.6.1-6.
